### ISSUE TITLE
Add support for disc number in albums

### ISSUE THREAD

**Issue by Kodmod** [SimpleMobileTools/Simple-Music-Player#333]
Currently the track numbers are repeated if an album has more than 1 disc. So it will show the following:
```
<1 (from disc 1)> <song name>
<1 (from disc 2)> <song name>
<2 (from disc 1)> <song name>
<2 (from disc 2)> <song name>
```
Instead, it should first show the tracks for disc 1 and then show the tracks for subsequent discs - with like a list header separating the different disc.

I'm loving this 'Simple' project. Great apps!

**Comment by ajgelado** [comment]
As of September 2022, albums with multiple discs are still sorted by track number (interleaving tracks from both discs). Tested with the most recent version, 5.14, in two different devices with Android 6 and 7, with identical results. The albums which cause the problem with Simple Music Player are ordered correctly in other players (i.e., Vinyl Music Player). The albums' tags have both the "Track" and "Position in Media Set" correctly set up (confirmed with a desktop ID3 editor). Can we reopen the issue?

**Comment by FWDekker** [comment]
@ajgelado This issue is closed because it's a duplicate of #331. It should be possible for @Kodmod to re-close this issue, this time as _not planned_ (a GitHub feature which did not exist when the issue was originally closed).

**Comment by ajgelado** [comment]
Yes, I saw that the other issue is still open after posting here. I have tried to delete my message, but in my browser, the options menu doesn't load.
